packages/vaadin-grid/src/vaadin-grid.d.ts

```diff
@@ -341,7 +341,7 @@ export interface GridEventMap<TItem> extends HTMLElementEventMap, GridElementEve
  * @fires {CustomEvent} loading-changed - Fired when the `loading` property changes.
  * @fires {CustomEvent} selected-items-changed - Fired when the `selectedItems` property changes.
  */
-declare class GridElement<TItem = GridDefaultItem> extends ScrollerElement {
+declare class GridElement<TItem = GridDefaultItem> extends PolymerElement {
```

Comment:
Let's use `HTMLElement` as in other components. I tried `yarn lint:types` and it works.

```suggestion
declare class GridElement<TItem = GridDefaultItem> extends HTMLElement {
```